model/metrics/container.yaml

```diff
@@ -0,0 +1,52 @@
+groups:
+  # container.cpu.* metrics and attribute group
+  - id: metric.container.cpu.time
+    type: metric
+    metric_name: container.cpu.time
+    brief: "Total CPU time consumed"
+    note: >
+      Total CPU time consumed by the specific container on all available CPU cores
+    instrument: counter
+    unit: "s"
+    attributes:
+      - ref: container.cpu.state
+        requirement_level: opt_in
```

Comment:
does it have to be opt-in? any harm in including it by default (i.e. `recommended`)?

Reply:
We would need to revisit this anyways I think, see https://github.com/open-telemetry/semantic-conventions/pull/282#discussion_r1533691935.

Also we decided to have this one optional for now because not all implementations can handle it in the same way: https://github.com/open-telemetry/semantic-conventions/pull/282#issuecomment-1909118310

Reply:
Mind creating an issue so it's not lost? 

Looking into https://github.com/open-telemetry/semantic-conventions/pull/282#issuecomment-1909118310 it sounds like `recommeded: if available` or `conditionally_required: if available` would work too.

Reply:
Sure, I have filed https://github.com/open-telemetry/semantic-conventions/issues/840